test/lib/BedrockTester.cpp

```diff
@@ -577,6 +577,12 @@ bool BedrockTester::readDB(const string& query, SQResult& result, bool online)
         command["Query"] = fixedQuery;
         command["Format"] = "JSON";
         auto row0 = SParseJSONObject(executeWaitMultipleData({command})[0].content)["rows"];
+        auto headerString = SParseJSONObject(executeWaitMultipleData({command})[0].content)["headers"];
```

Comment:
Maybe NAB but would it make sense to extract `SParseJSONObject(executeWaitMultipleData({command})[0].content)` into it's own variable so we're not calling the query twice in a row?

```cpp
JSON::Value content = SParseJSONObject(executeWaitMultipleData({command})[0].content);
auto row0 = content["rows"];
auto headerString = content["headers"];
```

Reply:
Yes, we should do that.